app/views_rails_6_1/layouts/application.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
do we still need this directory/file? it's identical to application.html.erb

Reply:
We don't need this file but ultimately when we want to make our `views_rails_6_1` files the default all we need to do is delete the current `views` folder and rename `views_rails_6_1` and everything inside should be in the right place.  If we delete it then I feel like it would be a game of catching these onesies and twosies and remember which ones we took out.